app/controllers/data_access_controller.rb

```diff
@@ -192,6 +197,17 @@ def options
 
   private
 
+  def echo_form_outdated?(access_config, methods)
+    return true if access_config.nil? || access_config.echoform_digest.nil? || methods.nil?
```

Comment:
It will set the defaults to `{service_options: nil}` which means that the 'Download' radio won't be checked, right? Or am I misreading that?